assignments/this.js

```diff
@@ -13,14 +13,50 @@
 
 // code example for Window Binding
 
-// Principle 2
+console.log(name);
```

Comment:
This example isn't referencing a window binding. You still need to use the `this` keyword ie. `this.name`. 

You can see a demonstration of a window binding by going into your developer tools in Chrome, accessing your console, and typing `this.name = "Alex"` and hitting enter. After, type `console.log(this)` and you'll see that what is logged is a Window object and, if you search through it (click on it), you will find a `name` property on it with a value of "Alex".